WhyNotWin11.au3

```diff
@@ -540,13 +541,25 @@ Func Main()
 			GUICtrlSetData($hCheck[8][2], _Translate($iMUI, "Disabled / Not Detected"))
 	EndSwitch
 
-	_SpaceCheck()
-	GUICtrlSetData($hCheck[9][2], @error & " GB " & $WINDOWS_DRIVE & @CRLF & @extended & " " & _Translate($iMUI, "Drive(s) Meet Requirements"))
-	If _SpaceCheck() Then
+	#Region - SpaceCheck
+	Local $iDiskSpace = _SpaceCheck(2)
+	Local $iPartitionSpace = _SpaceCheck(4)
+	Local $sSpaceResultFirstLine = _SpaceCheck(3) & " " & $iPartitionSpace & " GB, " & _Translate($iMUI, "Disk") & " " & _SpaceCheck(1) & ": " & $iDiskSpace & " GB"
+	; -- Check
+	If $iDiskSpace >= 64 And $iPartitionSpace >= 64 Then
+		; Partition and Disk >= 64 GB
 		_GUICtrlSetState($hCheck[9][0], $iPass)
-	Else
+		GUICtrlSetData($hCheck[9][2], $sSpaceResultFirstLine & @CRLF & _Translate($iMUI, "Partition and disk passed."))
```

Comment:
These strings also need to be added to translation file

Reply:
Is there.